tiledb/CMakeLists.txt

```diff
@@ -398,6 +393,9 @@ endif()
 # so we can use the targets created by the calls to find_package().
 add_library(TILEDB_CORE_OBJECTS_ILIB INTERFACE)
 
+# Link baseline library
+target_link_libraries(TILEDB_CORE_OBJECTS_ILIB INTERFACE baseline)
```

Comment:
The installation errors are coming from attaching the object library to the interface, which propagates dependency from baseline onto the interface.

It looks like TILEDB_CORE_OBJECTS is the right target to attach baseline to. Will also need it in TILEDB_CORE_OBJECTS_STATIC if so.

The CMake INTERFACE specifier is wrong here. PRIVATE is what you want, otherwise all the dependencies of the object libraries propagate upward.